core/src/main/java/org/apache/iceberg/deletes/PositionDeleteWriter.java

```diff
@@ -52,18 +53,30 @@ public PositionDeleteWriter(FileAppender<StructLike> appender, FileFormat format
     this.partition = partition;
     this.keyMetadata = keyMetadata != null ? keyMetadata.buffer() : null;
     this.delete = PositionDelete.create();
-    this.pathSet = CharSequenceSet.empty();
+    this.referencedDataFiles = CharSequenceSet.empty();
```

Comment:
@openinx @rdblue, it is the rename we discussed [here](https://github.com/apache/iceberg/pull/2945/files#r696236311).